handlers.ts

```diff
@@ -0,0 +1,61 @@
+import fs = require("fs");
+
+const filePath = "file.txt";
+
+
+async function getUpdateTime() {
```

Comment:
add async to function only if you use await in the function

Reply:
done